0000-interledger-eli5/0000-interledger-eli5.md

```diff
@@ -0,0 +1,124 @@
+
+5) The connector tries the password in the box, and of course - it opens.  The connector now knows this password will open the box the USD ledger manager has, and the ledger manager will put $10 USD into the connector's account on the USD ledger.  The connector trusts the USD ledger manager, and so gives the password to them.
+
+6) The USD ledger manager tries the password and when the box opens, puts the USD from the holding account into the connectors account.
+
+So how are things now?  The sender has no USD, that's good - those USD were meant to be sent away.  The receiver has ¥1050 JPY.  That's good, that's just what the sender wanted.  The payment is complete!
+
+_Note: The connector now has $10 USD, and no JPY. But that connector can now wait to be contacted by another sender, this time one from Japan, who wants to send JPY to the US.  The connector could then offer $1 USD for ¥109 JPY each.  When that payment is done, the connector will have ¥1090 JPY on the JPY ledger, a little bit more than the ¥1050 the connector had before the first payment.  That's one way connectors can earn money by linking senders and receviers on the Interledger._
+
+### The Magic of ILP: Payments Are An Illusion
+
+Something interesting about all of this, is that on the Interledger, it may well be correct to say that no assets are actually ever "sent" anywhere.  What actually happens is the various ledger managers involved all agree to write in their ledger books at about the same time.  What's written first is that the asset is _not_ in the senders account, and then that it _is_ in the sending ledgers holding account.  Then its written that the asset is _not_ in the connectors account, but _is_ in the receiving ledgers holding account.  When the password is received, the ledger managers do the same thing, but from the receiving ledgers holding account to the receivers account, and the sending ledgers holding account, to the connectors account.  Assets are appearing and disappearing _locally_ on different ledgers, and because this is happening at about the same time, and because everyone involved is happy with what's happening, it creates the _effect_ of an asset _seeming_ to be sent from the sender's account to the receivers account.  But really, it's an illusion.
+
+_Note: The reason for the "holding" part is that it protects the assets.  If anything goes wrong with the payment while the assets are on hold - for example, if the receiver rejects the payment - the assets can simply be taken from the holding accounts and put back into the accounts they came from.  In fact, in the case of the example given, if the receiver rejected the payment the assets would **have** to be put back.  Because if the receiver rejected the payment, no password would have been given that could open the boxes, so none of the ledger managers would have done anything._
+
+
+**Asset Transformation & Connectors**
```

Comment:
I wouldn't call this "transformation" but "exchange". The assets stay the same, but are exchanged along the way for other assets